### PR TITLE
Removed the rest of the fake nulls

### PR DESCRIPTION
**Pull request by tuxudo**

### CODE REVIEW

**Comment by bochoven** [comment]
I think I would prefer to get rid of the -987654 altogether. A migration could do it..

**Comment by tuxudo** [comment]
I thought I tested that migration, the others all were

**Comment by bochoven** [comment]
👍 just checking 😄

File: app/modules/fan_temps/fan_temps_model.php [hunk 1]
```diff
@@ -45,257 +45,257 @@ function __construct($serial='')
...
-		$this->rs['ts1s'] = -9876540; $this->rt['ts1s'] = 'FLOAT';
-		$this->rs['ts2p'] = -9876540; $this->rt['ts2p'] = 'FLOAT';
-		$this->rs['ts2s'] = -9876540; $this->rt['ts2s'] = 'FLOAT';
-		$this->rs['ts2v'] = -9876540; $this->rt['ts2v'] = 'FLOAT';
-		$this->rs['ttf0'] = -9876540; $this->rt['ttf0'] = 'FLOAT';
-		$this->rs['ttld'] = -9876540; $this->rt['ttld'] = 'FLOAT';
-		$this->rs['ttrd'] = -9876540; $this->rt['ttrd'] = 'FLOAT';
-		$this->rs['tw0p'] = -9876540; $this->rt['tw0p'] = 'FLOAT';
-		$this->rs['tw0p2'] = -9876540; $this->rt['tw0p2'] = 'FLOAT';
+		$this->rs['ta0p'] = 0; $this->rt['ta0p'] = 'FLOAT';
```
Comment (bochoven) [inline app/modules/fan_temps/fan_temps_model.php]: Just set the value to 0.0 and MR will use FLOAT, no need to specify that

File: app/modules/fan_temps/views/fan_temps_listing.php [hunk 2]
```diff
@@ -131,22 +131,22 @@
 			}
                 
 			var columnvar=$('td:eq(3)', nRow).html();
-			if(columnvar == "-9876540") {
+			if(columnvar == "-9876540" || columnvar == "") {
```
Comment (bochoven) [inline app/modules/fan_temps/views/fan_temps_listing.php]: If the migration succeeds, no need to check for `-9876540` anymore?

File: app/modules/munkireportinfo/views/munkireportinfo_tab.php [hunk 3]
```diff
@@ -45,7 +45,7 @@
 			$('#munkireportinfo-reportitems').text(data.reportitems); 
             
             // Format version
-            if (data.version != "-9876543" && (data.version)) {
+            if (data.version != "-9876543" && data.version != null && (data.version)) {
```
Comment (bochoven) [inline app/modules/munkireportinfo/views/munkireportinfo_tab.php]: Just check `data.version`, null is evaluated to FALSE See http://php.net/manual/en/language.types.boolean.php

File: app/modules/power/migrations/002_power_fix_indexes.php [hunk 4]
```diff
@@ -86,7 +86,7 @@ public function up()
             case 'mysql':
                 foreach($this->idx as $indexArray)
                 {
-                    $sql = 'DROP INDEX ' . $indexArray[0] . 'ON power';
+                    $sql = 'DROP INDEX ' . $indexArray[0] . ' ON power';
```
Comment (bochoven) [inline app/modules/power/migrations/002_power_fix_indexes.php]: This is already fixed

**Pull request merged**